src/main/java/com/epam/controller/ToursController.java

```diff
@@ -136,4 +135,12 @@ public ModelAndView updateTour(@RequestParam String tourId, @RequestParam String
     public String getAddTour() {
         return "addtour";
     }
+
+    @GetMapping("/updatetour")
+    public ModelAndView getUpdateTour(@RequestParam String idOfTour) {
```

Comment:
getUpdate?? What is it mean?

Reply:
We have 2 methods for 2 types of REST for this url, one for “GET” another for “POST”, cause of it we can’t have same named methods. “getUpdate” means that we use “GET” method on our page.

Reply:
you can use just updateTour, java supports it....

Reply:
Done